tests/psimulate/cluster/test_interface.py

```diff
@@ -1,38 +1,36 @@
+import socket
+
 import pytest
 
 from vivarium_cluster_tools.psimulate.cluster import validate_cluster_environment
-from vivarium_cluster_tools.psimulate.environment import ENV_VARIABLES
 
 
-@pytest.fixture(params=["user@int-uge-archive-p006"])
+@pytest.fixture(
+    params=[
+        "gen-slurm-sarchive-p0051",
+        "long-slurm-sarchive-p0022",
```

Comment:
what is the effect of these hard-coded nodes? are they only for testing?

Reply:
Yeah, these are just plausible strings for testing.